config/env/default.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUploadDir: './modules/users/client/img/profile/uploads/'
```

Comment:
@gustavodemari that comment is irrelevant anymore with the recent changes. Good job.